client/src/redux/reducers/userSlice.js

```diff
@@ -1,19 +1,78 @@
 import { createSlice } from '@reduxjs/toolkit'
+
+/**
+ * @property {trips}: [{
+ *  id: number, - Unique identifier for the trip, starts from 1 and auto-increments
```

Comment:
If user 1 creates trips 1,2,3, user 2 creates a trip, would it also have ID 1? Maybe a problem in the BE (work aroundable I think) 👀

Reply:
i dont think it matters, the id is so that the trip is unique. if user 2 starts creating then the id will be uniquely 4,5,... but most of the states will most likely be changed when we start connecting to backend.

Reply:
the user dont know what id their trips are assigned to. it is for the app to identify the trips